### PR TITLE
SUL23-70 | @jdwjdwjdw | Build and style the library branch pages

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- [SUL23-70](https://stanfordits.atlassian.net/browse/SUL23-70): FE-DEV | Build and style the branch pages

# Review By (Date)
- When convenient

# Steps to Test

1. Checkout branch or view [preview](https://deploy-preview-44--stanford-library.netlify.app/sample-content/david-rumsey-map-center)
2. Create library branch page
3. Confirm everything looks correct, including responsive
4. Review code ☕

[SUL23-70]: https://stanfordits.atlassian.net/browse/SUL23-70?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/nodes/node-sul-library.tsx [hunk 1]
```diff
@@ -1,67 +1,188 @@
...
+                          {isOpen ? 'Open' : 'Closed'}
+                        </div>
+                      }
+                    </div>
+                    {libraryPrimaryHours &&
+                      <div className="su-relative su-pb-70 md:su-pb-40">
+                        <label htmlFor="library-hours" className="su-sr-only">Choose a library</label>
+                        <select id="library-hours" className="su-absolute su-leading-none su-w-full su-text-black su-py-20 su-mb-20 su-rounded su-shadow su-type-1">
+                          {libraryPrimaryHours.map(date =>
+                              <option key={`library-hours-${date.day}`} selected={date.day === currentDay} disabled={date.day !== currentDay} value={date.day}>
```
Comment (pookmish) [inline src/components/nodes/node-sul-library.tsx:161]: I'm not sure if they want the options disabled or not. We'll check with them upon review.

File: src/components/nodes/node-sul-library.tsx [hunk 2]
```diff
@@ -1,67 +1,188 @@
...
+                        <div className="su-relative su-flex su-flex-row su-items-start su-mt-20 md:su-mt-18 su-mb-4 su-type-1">
+                          <ClockIcon width={19} className="md:su-absolute md:su-left-[-38px] md:su-top-01em su-mr-12"/>
+                          {isOpen ? 'Open' : 'Closed'}
+                        </div>
+                      }
+                    </div>
+                    {libraryPrimaryHours &&
+                      <div className="su-relative su-pb-70 md:su-pb-40">
+                        <label htmlFor="library-hours" className="su-sr-only">Choose a library</label>
+                        <select id="library-hours" className="su-absolute su-leading-none su-w-full su-text-black su-py-20 su-mb-20 su-rounded su-shadow su-type-1">
```
Comment (pookmish) [inline src/components/nodes/node-sul-library.tsx:159]: It's best to use `const inputId = useId();` for input ids. See the documentation https://beta.reactjs.org/reference/react/useId

**Review by pookmish — CHANGES_REQUESTED** [review]
This looks really great. Just one easy change needed and I'll get it merged in.

File: src/components/nodes/node-sul-library.tsx (hunk 2 above)
Comment (jdwjdwjdw) [reply src/components/nodes/node-sul-library.tsx]: Thanks - should be good now

**Review by pookmish — APPROVED** [review]

**Pull request merged**